example/src/pages/include1.md

```diff
@@ -1,4 +1,8 @@
-## Include 1 Page
+---
+import: true
```

Comment:
What happens when import is true?

Reply:
Ah, good catch. That was left over from something I was trying, and will probably come back to later, to identify which markdown files are imported into a parent or host markdown file so that they're slug is created from their parents slug, not their own directory location in the project. However, this is unnecessary at the moment because we create records for markdown files with imports from HTML, not from the markdown nodes. Sorry, the TL;DR answer is: Don't need it; I'll remove it!